### DISCUSSION TITLE
UI: see the picture while editing labels

### DISCUSSION THREAD

**Discussion by Tirallala** [photoprism/photoprism#2976] [Ideas]
When I edit a picture (tab details) there is a thumbnail of the picture on the left side. Can you please add this thumbnail also to tab labels? 

This would make checking and editing labels much more convenient, because you don't have to switch between tab details and labels constantly to see which picture you're editing at the moment.

As an example: I'm going through safari pictures and want to check if the animals are labeled properly. So I'm using the arrow navigation (top right in the edit picture view) going from picture to picture in tab labels. But I cannot check the labels because I don't which picture I'm currently working on without switching to the details tab
![details](https://user-images.githubusercontent.com/106486840/206284672-b1b70c5a-2edd-43de-8664-e39c6fa8b3fc.PNG)
![labels](https://user-images.githubusercontent.com/106486840/206284696-2f3b71a8-b404-4a10-807d-ab66a8f4436e.PNG)

**Comment by graciousgrey** [comment]
We already have a feature request for this: https://github.com/photoprism/photoprism/issues/628.
Feel free to leave a 👍 in there :)

**Comment by Tirallala** [comment]
sorry, I just checked the discussions and not the issues for similar issues. Thanks!
